pytest-selenium/tests/test_attribution_status.py

```diff
@@ -0,0 +1,33 @@
+from pages.content import Content
+from tests import markers
+
+
+@markers.test_case("C476303")
+@markers.parametrize("book_slug,page_slug", [("college-physics", "preface")])
+@markers.nondestructive
+def test_attribution_collapsed_by_default_expands_when_clicked(
+    selenium, base_url, book_slug, page_slug
+):
+
+    """
+    attribution section is initially collapsed, expands when clicked
+
+    GIVEN: A page URL in the format of {base_url}/books/{book_slug}/pages/{page_slug}
+    WHEN: The page is fully loaded
+    THEN: The attribution section is collapsed by default
+    AND The attribution section opens when clicked
+    """
+    content = Content(selenium, base_url, book_slug=book_slug, page_slug=page_slug).open()
+    attribution = content.attribution
+
+    assert not attribution.is_open
+
+    attribution.click_attribution_link()
+
+    assert attribution.is_open
+
+    attribution.click_attribution_link()
+    # """click on attribution again and verify it closes
```

Comment:
You are using a docstring here which should only come after a `def` or `class` definition. comments within code use a `# `

Reply:
@m1yag1  I updated the comments and docstrings

Reply:
What's up with line 26. I still see a comment with `"""` inside the code

Reply:
@m1yag1  Thats my bad....I updated this